pull-request-practice.py

```diff
@@ -0,0 +1,6 @@
+def review():
+    print("これはコードレビューのサンプルです")
```

Comment:
```suggestion
    print('これはコードレビューのサンプルです')
```

チームのルールとして文字列の引用符はシングルクォーテーションでお願いします。